src/System.Device.Gpio/System/Device/Pwm/Channels/PwmChannel.Linux.cs

```diff
@@ -0,0 +1,31 @@
+﻿// Licensed to the .NET Foundation under one or more agreements.
+// The .NET Foundation licenses this file to you under the MIT license.
+// See the LICENSE file in the project root for more information.
+
+namespace System.Device.Pwm
+{
+    public partial class PwmChannel
+    {
+        /// <summary>
+        /// Creates a PWM channel running on Unix.
+        /// </summary>
+        /// <param name="pwmChip">The PWM chip number.</param>
+        /// <param name="pwmChannel">The PWM channel number.</param>
```

Comment:
Should `pwmChip` and `pwmChannel` just be named `chip` and `channel`?  I recall @JeremyKuhne and @bartonjs suggesting `pinNumber` for GPIO pin be renamed to just `pin` for simplicity.